src/lsg/patterns/view-switch/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { colors } from '../colors';
+import { Size } from '../copy';
+import { Icon, IconName, IconProps, Size as IconSize } from '../icons';
+import * as React from 'react';
+import { getSpace, Size as SpaceSize } from '../space';
+import styled from 'styled-components';
+
+export interface ViewSwitchProps {
+	fontSize?: Size;
+	justify?: 'start' | 'center' | 'end' | 'stretch';
+	leftVisible: boolean;
+	onLeftClick: React.MouseEventHandler<SVGElement>;
+	onRightClick: React.MouseEventHandler<SVGElement>;
+	rightVisible: boolean;
+	title: string;
+}
+
+interface StyledIconProps extends IconProps {
+	visible: boolean;
+}
+
+interface StyledViewSwitchProps {
+	fontSize?: Size;
+	justify?: 'start' | 'center' | 'end' | 'stretch';
+}
+
+const StyledViewSwitch = styled.div`
+	display: inline-flex;
```

Comment:
doesn't have to be `inline-flex`, `flex` only is enough

Reply:
actually in this case you're right. ✅